test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractHTSSuite.java

```diff
@@ -478,6 +481,85 @@ private HapiApiSpec tokenTransferFromFeeCollector() {
 				);
 	}
 
+	private HapiApiSpec hbarTransferFromFeeCollector() {
+		return defaultHapiSpec("HbarTransferFromFeeCollector")
+				.given(
+						cryptoCreate(ACCOUNT).balance(10 * ONE_HUNDRED_HBARS).maxAutomaticTokenAssociations(10),
+						cryptoCreate(RECEIVER).maxAutomaticTokenAssociations(10),
+						cryptoCreate(TOKEN_TREASURY),
+						cryptoCreate(FEE_COLLECTOR),
```

Comment:
We should create the FEE_COLLECTOR account with intial balance of 0, this way we can assert it has a balance ot 100 at the end of the test; 1000000100 is not very clear